sandbox/public/index.html

```diff
@@ -51,10 +52,19 @@
 
       alloy("event", {
         viewStart: true,
+        xdm: {
+          "_myorg": {
```

Comment:
So this is required right? :(

Reply:
For data related to customer-defined mixins, yeah. :/